### PR TITLE
class globalException and validation annotations were added

### PR DESCRIPTION
**Pull request by vgladkyi**

### CODE REVIEW

File: src/main/java/bookstore/dto/BookRequestDto.java [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
 
 @Data
 public class BookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
     private String isbn;
+    @Min(0)
```
Comment (okuzan) [inline src/main/java/bookstore/dto/BookRequestDto.java:16]: Min won't work for BigDecimal. Use @Positive or @DecimalMin

File: src/main/java/bookstore/dto/BookRequestDto.java [hunk 2]
```diff
@@ -1,13 +1,19 @@
 package bookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class BookRequestDto {
+    @NotNull
```
Comment (okuzan) [inline src/main/java/bookstore/dto/BookRequestDto.java]: Let's ensure String's cant be like this "" or " "

**Review by okuzan — CHANGES_REQUESTED** [review]

File: src/main/java/bookstore/dto/BookRequestDto.java (hunk 1 above)
Comment (Elena-Bruyako) [reply src/main/java/bookstore/dto/BookRequestDto.java:16]: Not fixed

**Review by Elena-Bruyako — APPROVED** [review]
Approve, but change validation annotation according to the mentors comment

**Pull request merged**